a/antsan-i-kristy.md

```diff
@@ -0,0 +1,5 @@
+# Antsan'i Kristy
+
+![antsan-i-Kristy](antsan-i-Kristy.jpg)
```

Comment:
This image filename needs to be a lowercase K.
Please use the `git mv` trick to rename it.